core/nginx/conf/nginx.conf

```diff
@@ -33,6 +33,17 @@ http {
       default $http_x_forwarded_proto;
       ''      $scheme;
     }
+    map $uri $expires {
```

Comment:
Let's keep caching out of this PR maybe? it goes hand-in-hand with @Diman0's work on moving /static out ...

and as is, I am not convinced that it won't break things (.js may be dynamic on webmails)

Reply:
I'd like to set the caching header in one place, but maybe it's better to set the headers in the backend containers (admin, webmail, ...). This way users of "other" proxy setups will have headers too.
I'll triple-check to see if proxied content is handled correctly (ie. upstreams caching and vary headers are used).
And I'll try to add the headers via gunicorn/flask/static handler (and maybe even use your snipped for serving pre-compressed content)

Reply:
@nextgens Okay, I've modified the caching to work only on mailu static content (admin and front container).
And I've enabled caching headers in the roundcube container (enabled apache modules necessary).